all-components/image.css

```diff
@@ -0,0 +1,34 @@
+@import url('./utilities.css');
+
+
+.res-img {
+    width: 100%;
+    height: auto;
+}
+
+.round-img {
+    object-fit: cover;
+    border-radius: 50%;
+}
```

Comment:
can use apsect-ratio: 1/1 to avoid elliptical image

Reply:
Will update, thanks for your suggestion.